From: Peter Conrad <peter.conrad@uni-jena.de>
Date: Thu, 17 Dec 2020 15:00:52 +0100
Subject: [PATCH] declare yylloc as extern

---
 dtc-lexer.l | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dtc-lexer.l b/dtc-lexer.l
index 3b41bfca636c..9b9c29e6f311 100644
--- a/dtc-lexer.l
+++ b/dtc-lexer.l
@@ -39,7 +39,7 @@ LINECOMMENT	"//".*\n
 #include "srcpos.h"
 #include "dtc-parser.tab.h"
 
-YYLTYPE yylloc;
+extern YYLTYPE yylloc;
 
 /* CAUTION: this will stop working if we ever use yyless() or yyunput() */
 #define	YY_USER_ACTION \
